src/Core/Extensible.php

```diff
@@ -124,7 +124,7 @@ protected function defineExtensionMethods()
         foreach ($extensions as $extensionClass => $extensionInstance) {
             foreach ($this->findMethodsFrom($extensionInstance) as $method) {
                 $this->addCallbackMethod($method, function ($inst, $args) use ($method, $extensionClass) {
-                    /** @var Extensible $inst */
+                    /** @var self $inst */
```

Comment:
Why this change? Does the IDE not pick up on the methods if with `Extensible`, but it does with `self`?

Reply:
Honestly I don't know, I've reverted it